frontend/src/features/mapSideBar/containers/MotiInventoryContainer.test.tsx

```diff
@@ -470,58 +488,52 @@ describe('MotiInventoryContainer functionality', () => {
     beforeEach(() => {
       history.push('/mapview?sidebar=true&sidebarContext=addBareLand');
       isPidAvailable.mockResolvedValue({ available: false });
+      process.env.REACT_APP_TENANT = 'MOTI';
       findByPid.mockResolvedValueOnce(mockParcelLayerResponse);
     });
 
     it('displays the duplicate pid modal if a duplicate pid is detected.', async () => {
       const { container, getByTestId } = renderContainer({});
-      await fillInput(container, 'searchPid', '123-456-789');
-      act(() => {
-        fireEvent.click(getByTestId('pid-search-button'));
-      });
+      const searchPid = container.querySelector('input[name="searchPid"]');
+      await waitFor(() => fireEvent.change(searchPid!, { target: { value: '123-456-789' } }));
+      fireEvent.click(getByTestId('pid-search-button'));
       const modal = await screen.findByText(/The parcel identifier \(PID\) 123-456-789/g);
       expect(modal).toBeInTheDocument();
     });
     it('does not display the modal text if there is no duplicate pid', async () => {
       isPidAvailable.mockResolvedValue({ available: true });
       const { container, getByTestId } = renderContainer({});
-      await fillInput(container, 'searchPid', '123-456-789');
-      act(() => {
-        fireEvent.click(getByTestId('pid-search-button'));
-      });
+      const searchPid = container.querySelector('input[name="searchPid"]');
+      await waitFor(() => fireEvent.change(searchPid!, { target: { value: '123-456-789' } }));
```

Comment:
this is interesting did we add this as a result of a warning? waitFors are normally used to essentially wait for an async page change that is required by some logic. However, there shouldn't be a page change required for changing the searchPid field is there? Does this test work without that waitFor?